Testsuite: oracle.json.tests.soda.test_OracleCollectionAdmin
Tests run: 9, Failures: 0, Errors: 1, Time elapsed: 4.855 sec

Testcase: testCreateIndexTTL(oracle.json.tests.soda.test_OracleCollectionAdmin):	Caused an ERROR
null
oracle.soda.rdbms.impl.SODAUtils$1OracleSQLException
	at oracle.soda.rdbms.impl.SODAUtils.makeExceptionWithSQLText(SODAUtils.java:124)
	at oracle.soda.rdbms.impl.SODAUtils.makeExceptionWithSQLText(SODAUtils.java:87)
	at oracle.soda.rdbms.impl.OracleCollectionImpl.callCreateIndexPLSQL(OracleCollectionImpl.java:1789)
	at oracle.soda.rdbms.impl.OracleCollectionImpl.createIndex(OracleCollectionImpl.java:1959)
	at oracle.soda.rdbms.impl.OracleCollectionImpl.access$900(OracleCollectionImpl.java:69)
	at oracle.soda.rdbms.impl.OracleCollectionImpl$OracleCollectionAdministrationImpl.createIndex(OracleCollectionImpl.java:2398)
	at oracle.json.tests.soda.test_OracleCollectionAdmin.testCreateIndexTTL(test_OracleCollectionAdmin.java:2300)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at oracle.json.testharness.JsonTestCase.run(JsonTestCase.java:45)
Caused by: java.sql.SQLException: ORA-27486: insufficient privileges

https://docs.oracle.com/error-help/db/ora-27486/
	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:702)
	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:608)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1277)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:1102)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:456)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:482)
	at oracle.jdbc.driver.T4CCallableStatement.doOall8(T4CCallableStatement.java:216)
	at oracle.jdbc.driver.T4CCallableStatement.executeForRows(T4CCallableStatement.java:1164)
	at oracle.jdbc.driver.OracleStatement.executeSQLStatement(OracleStatement.java:1652)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1417)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3731)
	at oracle.jdbc.driver.OraclePreparedStatement.execute(OraclePreparedStatement.java:3945)
	at oracle.jdbc.driver.OracleCallableStatement.execute(OracleCallableStatement.java:4202)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.execute(OraclePreparedStatementWrapper.java:1011)
	at oracle.soda.rdbms.impl.OracleCollectionImpl.callCreateIndexPLSQL(OracleCollectionImpl.java:1771)
Caused by: Error : 27486, Position : 0, SQL = begin
  DBMS_SODA_ADMIN.CREATE_INDEX(
                   P_URI_NAME   => :1 ,
                   P_INDEX_SPEC => :2 );
end;, Original SQL = begin
  DBMS_SODA_ADMIN.CREATE_INDEX(
                   P_URI_NAME   => ?,
                   P_INDEX_SPEC => ?);
end;, Error Message = ORA-27486: insufficient privileges

	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:710)


